### PR TITLE
nfs4: add tests for readdir/setclientid records v2

### PR DESCRIPTION
**Pull request by ghostinthehive**
Previous PR: #658
S-V for unittests at https://github.com/OISF/suricata/pull/6819/commits/c4bd3cd70e3b0a2718eb404a214748780cfc4db4

### CODE REVIEW

File: tests/nfs4-response-readdir/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+PCAP
+====
+
+PCAP by Victor Julian
```
Comment (victorjulien) [inline tests/nfs4-response-readdir/README.md:4]: typo

File: tests/nfs4-setclientid/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+PCAP
+====
+
+PCAP by Victor Julian
```
Comment (victorjulien) [inline tests/nfs4-setclientid/README.md:4]: typo

File: tests/nfs4-setclientid/test.yaml [hunk 3]
```diff
@@ -0,0 +1,84 @@
+args:
+- -k none
+
+checks:
```
Comment (victorjulien) [inline tests/nfs4-setclientid/test.yaml:4]: no nfs checks?

File: tests/nfs4-response-readdir/test.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+requires:
+  features:
+    - RUST
+  files:
+    - rust/src/nfs/nfs4.rs
+
+args:
+- --set stream.midstream=true
+
+checks:
```
Comment (victorjulien) [inline tests/nfs4-response-readdir/test.yaml:10]: no nfs checks?

**Review by victorjulien — CHANGES_REQUESTED** [review]
see inline comments

**Comment by ghostinthehive** [comment]
> see inline comments

@victorjulien Please correct me If I'm wrong, I can see only `NFSPROC4_WRITE` being logged, all the rest is logged as `malformed_data` so I guess it means they are invalid records?

Which left me with no choice but logging these PCAPs as just `flow` events, So I'm beginning to question the need of having S-V tests for these two cases?
Or am I missing something here?

**Comment by catenacyber** [comment]
@ghostinthehive what is the status here ?

**Comment by catenacyber** [comment]
Closing as unfinished

**Pull request closed without merging**